.github/workflows/ipfs-deploy.yml

```diff
@@ -57,3 +57,11 @@ jobs:
 
           docker tag $IMAGE_TAG $IMAGE_ID:$VERSION
           docker push $IMAGE_ID:$VERSION
+
+					# Make image reference available to subsequent steps
+					echo "IMAGE_RELEASE_ID=$(echo $IMAGE_ID:$VERSION)" >> $GITHUB_ENV
```

Comment:
Indentation is broken again here.  You may want to verify you have EditorConfig extension for your IDE, as that should use tabs everywhere except YAML, which should be spaces (because YAML is dumb).

Reply:
Oops, looks like editorconfig wasn't setup properly.  https://github.com/DarkFlorist/lunaria/pull/256